cms/wizards/views.py

```diff
@@ -144,8 +147,7 @@ def done(self, form_list, **kwargs):
                     url = '/'
             else:
                 url = '/'
-
-        return SimpleTemplateResponse("cms/wizards/done.html", {"url": url})
+        return HttpResponse(MODAL_HTML_REDIRECT.format(url=escape(url)))
```

Comment:
Should we write a test for this? @fsbraun ?

Reply:
I added a test - quite a thing for these two-step wizards. Actually, it is good to have a test.